docs/user/whats-new.asciidoc

```diff
@@ -1,128 +1,130 @@
 
-image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt0957c51279df4bec/67c23ac28c6f4e7a59b7f3c6/file-uploader-enhancements-8.18.gif[Uploading multiple files with the file uploader]
+This context-aware rule creation shortcut helps you improve monitoring while staying informed by data. 
+
+image::images/alerting_rules_from_visualizations.png[One Click Monitoring: Create Alerting rules from visualizations]
 
 [discrete]
-=== Alerting, cases, and connectors
+==== Alerts panel in dashboards
+
+You can now include Alerts tables as panels in dashboards. Alerts panels provide real-time visibility into system-triggered alerts alongside other visualizations. You can correlate alerts with other dashboard components, such as logs, metrics, Discover sessions, or Lens visualizations, without needing to switch contexts.
+
+image::images/alerts_panel_dashboard.png[Alerts panel in Dashboards]
 
 [discrete]
-==== Case templates are Generally Available
+==== Alerts history clean-up available in Technical Preview
 
-Case templates are now Generally Available. Case templates provide useful starting points for investigations in both Security and Observability. Add templates in Case Settings and leverage them as a starting point in any new case.
+You can now manually trigger alerts deletion in bulk. This new feature starts a cleanup task that runs asynchronously in the background. You can specify a threshold for alert age or duration and choose to delete active alerts, inactive alerts, or both. This feature is currently only available under Stack Management.
 
-image::images/add-template-flyout.png[The Add template flyout in Case Settings, width=90%]
+image::images/cleanup_alerts.png[Alerts history clean-up available in Technical Preview]
 
 [discrete]
-==== Case observables 
+==== Maintenance windows public APIs
 
-Case observables enable structured data collection. You can now add common observables to any case out of the box and extend the types of observable case data to include custom options.
+Maintenance windows now have public HTTP APIs, enabling programmatic management. You can leverage these APIs to perform Create, Read, Update, and Delete (CRUD) operations on maintenance windows, giving you greater control.
 
 [discrete]
-==== Certified for ServiceNow Xanadu release
 
-The Service Now connector is certified for the Xanadu release. 
+You can now explore Cases data in dashboards and Discover. You can use the new Cases analytics index to build dashboards and visualizations of Cases activities and trends. These data points offer improved visibility into incident management efficiency in Kibana for operation managers.
```

Comment:
Need to pull this. Was removed from 8.19/9.1.

cc: @michaelolo24 @paulewing @tiamliu @adcoelho 

```suggestion
```